docs/test/test-explorer-faq.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Test Explorer FAQ | Microsoft Docs"
+ms.date: "1/15/2018"
+ms.reviewer: ""
+ms.suite: ""
+ms.technology: visual studio
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+helpviewer_keywords: 
+  - "Test Explorer"
+  - "Test window"
+  - "Visual Studio Test Explorer"
+  - "summary line"
+  - "unit tests"
+  - "Test Explorer FAQ"
+ms.author: "kehavens"
+ms.workload: 
+  - "multiple"
+author: "kendrahavens"
+---
+# Visual Studio Test Explorer FAQ
+
+## Test Discovery
+
+### 1. The Test Explorer is not discovering my tests that have theories, custom adapters, custom traits, use #ifdefs, or are dynamically defined. How can I discover these tests?
+
+  Build your project and make sure assembly based discovery is turned on in **Tools > Options > Test**.
+
+  [Real Time Test Discovery](https://go.microsoft.com/fwlink/?linkid=862824), which is source-based test discovery, can’t discover some tests in cases such as theories, custom adapters, custom traits, use #ifdefs, or that are dynamically defined in some other way. A build is required for those tests to be accurately discovered. In the 15.6 Previews assembly-based discovery (the traditional discoverer) will run only after builds. This means Real Time Test Discovery will discover as many tests as it can as you are typing in between builds, but still allow theories (or any dynamically defined tests) to appear after a build. This will improve the responsiveness but still allow you to get complete and accurate results after a build.
+
+### 2. What does the '+' (plus) symbol that appears in the top line of the Test Explorer mean?
+
+  The '+' (plus) symbol indicates more tests may be discovered on build if discovering tests from built assemblies is on. This will appear if dynamically defined tests are detected in your project.
```

Comment:
Perhaps phrase this in a way that's consistent with the previous item: "...more tests may be discovered after a build is assembly-based discovery is turned on."